var/spack/repos/builtin/packages/curl/package.py

```diff
@@ -88,13 +95,19 @@ class Curl(AutotoolsPackage):
     variant("librtmp", default=False, description="enable Rtmp support")
     variant("ldap", default=False, description="enable ldap support")
     variant("libidn2", default=False, description="enable libidn2 support")
-    variant(
-        "libs",
-        default="shared,static",
-        values=("shared", "static"),
-        multi=True,
-        description="Build shared libs, static libs or both",
-    )
+    for plat in ["darwin", "cray", "linux"]:
+        with when("platform=%s" % plat):
+            variant(
+                "libs",
+                default="shared,static",
+                values=("shared", "static"),
+                multi=True,
+                description="Build shared libs, static libs or both",
+            )
+            # curl queries pkgconfig for openssl compilation flags
+            depends_on("pkgconfig", type="build")
+
+    variant("shared", default=False, description="enable building shared libraries")
```

Comment:
This and the `libs` variant are too similar, and their descriptions indicate that this variant is redundant. Furthermore, the NMakeBuilder doesn't appear to use this varian (it always builds in one way), so perhaps it could just be removed?

Reply:
Sounds good. Will amend.

Reply:
Amended.